OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -70,88 +74,123 @@ public void InvokeCommand(string name, string arg)
 				return;
 			}
 
-			switch (name)
+			if (!commandHandlers.TryGetValue(name, out var command))
+				return;
```

Comment:
Sorry, one last style nit that I missed last time: Having an early return guarding a single line isn't great code style. Can we please change instead to


```csharp
if (commandHandlers.TryGetValue(name, out var command))
	command.Handler(arg, world);
```

and likewise in the other file.

Reply:
Changed it.